tests/robustness/powerfailure_test.go

```diff
@@ -140,7 +141,8 @@ func doPowerFailure(t *testing.T, du time.Duration, fsMountOpt string, useFailpo
 
 	select {
 	case <-time.After(10 * time.Second):
-		t.Error("bbolt should stop with panic in seconds")
+		t.Log("bbolt should stop with panic in 10 seconds")
+		t.Log("switch to kill")
```

Comment:
Probably we should fail the test in such case, because it doesn't run as expected.
```suggestion
		t.Log("bbolt is supposed to be already stopped, but actually not yet; forcibly kill it")
```

Reply:
Hard to tell. If that storage is too fast, before we invoke failpoint, db size has been resized and the size is large enough.